tests/xsc_test.go

```diff
@@ -0,0 +1,95 @@
+package tests
```

Comment:
if a get this right all this test are currently unit tests and they can sit near the xsc service code and no neet the XSC testing suite

Reply:
in the client i've put just a simple unit test to make sure things don't break